### ISSUE TITLE
how can i train with modelnet40?

### ISSUE THREAD

**Issue by heebinYoo** [luca310795/tensor-field-networks#1]
modelnet10 is work!

However, I change train.py like this

NUM_CLASSES = 40
TRAIN_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="train", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)
TEST_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="test", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)

but loss is diverge :( 
so sadddd......

can you let me know setup for modelnet 40?

**Comment by heebinYoo** [comment]
I find it. 
HAPPY~

fix it as followings.


train.py 

NUM_CLASSES = 40
TRAIN_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="train", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)
TEST_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="test", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)

parser.add_argument("--model", default="tfn_8_channels", help="Model name [default: tfn_4_channels]")

eval.py
NUM_CLASSES = 40
SHAPE_NAMES = [line.rstrip() for line in \
    open(os.path.join(ROOT_DIR, "modelnet/data/modelnet40_normal_resampled/modelnet40_shape_names.txt"))]
TRAIN_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="train", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)
TEST_DATASET = dataloader.ModelNetDataloader(root=DATA_PATH, npoints=NUM_POINT, split="test", modelnet10=False, normal_channel=False, batch_size=BATCH_SIZE)



models/tfn_xxxx.py
NUM_CLASSES = 40

**Comment by heebinYoo** [comment]
done
